packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts

```diff
@@ -45,6 +46,30 @@ describe('JSONSchema generator', () => {
       });
     });
 
+    describe('when used with a schema with a string and uuid as format', () => {
+      const schema: JSONSchema = {
+        type: 'object',
+        properties: {
+          id: { type: 'string', format: 'uuid' },
```

Comment:
I believe it currently passes only because regex misses `^`.

Reply:
^ added now :)